templates/events/calendar.html

```diff
@@ -58,20 +58,20 @@ <h1 class="text-center">{{title}}
       <div class="form-check form-check-inline"><h4>Show</h4></div>
       <div class="form-check form-check-inline">
         <input type="checkbox" class="form-check-input" id="materialInline1" {% if "approved" in select_status %}checked{%endif%}>
-        <label class="form-check-label" for="materialInline1">Approved</label>
-      </div>
-      <div class="form-check form-check-inline">
-          <input type="checkbox" class="form-check-input" id="materialInline2" {% if "disapproved" in select_status %}checked{%endif%}>
-          <label class="form-check-label" for="materialInline2">Disapproved</label>
+        <label class="form-check-label" for="materialInline1">Pending</label>
       </div>
+{#      <div class="form-check form-check-inline">#}
+{#          <input type="checkbox" class="form-check-input" id="materialInline2" {% if "disapproved" in select_status %}checked{%endif%}>#}
+{#          <label class="form-check-label" for="materialInline2">Disapproved</label>#}
+{#      </div>#}
       <div class="form-check form-check-inline">
```

Comment:
I think the "published" checkbox should be selected by default. Thanks.

Reply:
i double checked the user event, it seems user event does not have the published checked by default.

Reply:
This has been fixed by Bing.